esmvaltool/preprocessor/_time.py

```diff
@@ -192,6 +199,73 @@ def spans_three_months(time):
     return cube.extract(three_months_bound)
 
 
+def regrid_time(cube, frequency):
+    """
+    Align time axis for cubes so they can be subtracted.
+
+    Operations on time units, calendars, time points and auxiliary
+    coordinates so that any cube from cubes can be subtracted from any
+    other cube from cubes. Currently this function supports only monthly
+    and daily data time frequencies, but it will be adapted for sub-day
+    frequencies in the future. Frequency is passed in straight from
```

Comment:
Remove the information 'Frequency is passed in straight from ..`, this is relevant only if using the function from the recipe, not when you use it as a python function.

Reply:
Also, remove the ', but it will be adapted', we can see about that when you actually do it.